src/feature/home-page/home-page-details/home-page-details.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useMemo } from 'react';
+import { useLoaderData, useOutletContext } from 'react-router-dom';
+import { ErrorComponent } from '@lib';
+import {
+  getPeopleFormatted,
+  People,
+  PeopleFormatted,
+  PeopleUnknown,
+} from '@utils';
+import './home-page-details.css';
+
+export type HomePageDetailsProps = {
+  closeFn: () => void;
+};
+
+const errorMessage = "It seems we don't know this person";
+
+export const HomePageDetails: FC = () => {
+  const { closeFn } = useOutletContext<HomePageDetailsProps>();
+
+  const item = useLoaderData<People | PeopleUnknown>();
+
+  const itemFormatted: PeopleFormatted | null = useMemo(() => {
+    return 'url' in item === false ? null : getPeopleFormatted(item, true);
+  }, [item]);
+
+  return (
+    <div>
+      <button type="button" onClick={() => closeFn()}>
+        x
+      </button>
```

Comment:
```suggestion
      <button type="button" onClick={() => closeFn()}>
        <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" stroke="#fff">
          <line x1="4" y1="4" x2="20" y2="20" />
          <line x1="4" y1="20" x2="20" y2="4" />
        </svg>
      </button>
```
What about to starting using graphics?

Reply:
I'll think about that, thanks :)